routing/matcher.go

```diff
@@ -96,7 +90,7 @@ func (err *definitionError) Error() string {
 }
 
 // rx identifying the 'free form' wildcards at the end of the paths
-var freeWildcardRx = regexp.MustCompile("/[*][^/]+$")
+var freeWildcardRx = regexp.MustCompile("/[*][^/]*$")
```

Comment:
This means, that `Path("/*")` is fine (unnamed wildcards), right?

Reply:
After the refactoring, this is used in:
* `addSubtreeLeafsToPath` for removing the free wildcard param from the end
* `extractWildcardParamNames` to see if a `Path` predicate ends with a free wildcard param
* `hasFreeWildcardParam`  to see if a `Path` predicate ends with a free wildcard param

The code using this regexp cares for both named and unnamed free wildcard params.

Reply:
thanks